components/search-bar/_search-bar.scss

```diff
@@ -34,12 +35,22 @@ $inputHeight: 38px;
         }
       }
     }
+
     &.not-empty{
       .sdc-input__input {
         border-color: $blue;
+
```

Comment:
why not 
.search-button { 
.....
   &:hover {
   }
}

Reply:
Because it covered SPECIAL case when input is not empty